distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -2301,6 +2301,18 @@ Event event = new Event("ddf/notification/catalog/downloads", properties);
 eventAdmin.postEvent(event);
 ----
 
+=== Configuring system.properties
+
+The `system.properties` file found under `$DDF_HOME/etc` contains properties that will be made available through system properties at the very beginning of Karaf's boot process. This file can be used to change specific system-level properties such as thread-pool count and default directory locations.
+
+==== Thread Pool
+
+The `org.codice.ddf.system.threadPoolSize` property can be used to specify the size of the thread pool used by the federation strategy for handling queries.
+
+* By default, this value is set to 128.
```

Comment:
Don't know if this is always true and even if it were, 128 would probably be too high for most systems. Until we have profiled the system with different thread pool sizes, I don't think I would recommend a specific value.